src/FS.Embedded/Manifest/EmbeddedFilesManifest.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Extensions.FileProviders.Embedded.Manifest
+{
+    internal class EmbeddedFilesManifest
+    {
+        private static readonly char[] _invalidFileNameChars = Path.GetInvalidFileNameChars()
```

Comment:
Looks similar to the result of  `Path.GetInvalidPathChars()`. Could you just use that instead?

Reply:
This follows whatever physicalfileprovider does, I could use what you are suggesting, but I don't know if there is a good reason for it. Maybe @Tratcher knows? See https://github.com/aspnet/FileSystem/blob/dev/src/FS.Physical/Internal/PathUtils.cs#L12

Reply:
I don't. That code has been refactored a few times since I looked at it last.

Reply:
```
Path.GetInvalidFileNameChars().Except(Path.GetInvalidPathChars())
ExceptIterator { ':', '*', '?', '\\', '/' }
```
So I think this should stay as it is.